src/port1.0/portsandbox.tcl

```diff
@@ -82,6 +83,16 @@ proc portsandbox::set_profile {target} {
     if {${configure.ccache}} {
         lappend allow_dirs $ccache_dir
     }
+    if {${build.type} eq "xcode"} {
+        # whitelist directories used by Xcode tools
+        lappend allow_dirs $portdbpath/home/Library/Developer/Xcode
+        lappend allow_dirs $portdbpath/home/Library/Caches
+        lappend allow_dirs $portdbpath/home/Library/org.swift.swiftpm
+        lappend allow_dirs $portdbpath/home/.swiftpm
```

Comment:
This could all be one lappend call (still across multiple lines for readability).

Reply:
Done.